test/jasmine/tests/transform_sort_test.js

```diff
@@ -283,7 +283,7 @@ describe('Test sort transform interactions:', function() {
 
         function wait() {
             return new Promise(function(resolve) {
-                setTimeout(resolve, 60);
+                setTimeout(resolve, 100);
```

Comment:
@etpinard it took 3 tries to get this test to pass on CI with the original timeout, first try with it a little longer.